pkg/aggregate/aggregate_test.go

```diff
@@ -1,20 +1,81 @@
+// +build unit
+
 package aggregate
 
 import (
 	"fmt"
 	"testing"
 )
 
-func TestAggragators(t *testing.T) {
-	aggrString := "*"
-	aggr, err := AggrsFromString(aggrString)
-	if err != nil {
-		t.Fatal(err)
+func TestAggregators(t *testing.T) {
+	testCases := []struct {
+		desc               string
+		aggString          string
+		data               map[int64]float64
+		exprCol            string
+		bucket             int
+		expectedUpdateExpr string
+		expectedSetExpr    string
+		expectFail         bool
+	}{
+		{"Should aggregate data with Count aggregator", "count", map[int64]float64{1: 7.5, 2: 2.5}, "v", 1,
+			"_v_count[1]=_v_count[1]+2;", "_v_count[1]=2;", false},
+
+		{"Should aggregate data with Sum aggregator", "sum", map[int64]float64{1: 7.5, 2: 2.5}, "v", 1,
+			fmt.Sprintf("_v_sum[1]=_v_sum[1]+%f;", 10.0),
+			fmt.Sprintf("_v_sum[1]=%f;", 10.0), false},
+
+		{"Should aggregate data with Sqr aggregator", "sqr", map[int64]float64{1: 2.0}, "v", 1,
+			fmt.Sprintf("_v_sqr[1]=_v_sqr[1]+%f;", 4.0),
+			fmt.Sprintf("_v_sqr[1]=%f;", 4.0), false},
+
+		// todo: enable when bug is fixed - IG-8675
```

Comment:
Shouldn't be needed anymore with proper ignores, but generally speaking `todo`->`TODO` 👷